components/DailyInfo.tsx

```diff
@@ -0,0 +1,149 @@
+import { zonedTimeToUtc } from "date-fns-tz";
+import React from "react";
+import SVG from 'react-inlinesvg'
+
+const PT = "US/Pacific";
```

Comment:
Moving this in the `constants` file can be useful so we can share the timezone between this file and the Guerilla one !